src/pudl_archiver/depositors/zenodo.py

```diff
@@ -122,7 +125,7 @@ async def create_deposition(self, metadata: DepositionMetadata) -> Deposition:
         Returns:
             The brand new deposition - unpublished and with no files in it.
         """
-        url = f"{self.api_root}/deposit/depositions"
+        url = f"{self.api_root}/records"
```

Comment:
To do: add `records` to the `api_root` rather than specifying it, since every endpoint seems to refer to it.